libopenage/datastructure/constexpr_map.h

```
@@ -268,15 +238,13 @@ class ConstMap<K, V, 0> {
 	 * Fetching a value from the end element aborts as it has no entry.
 	 */
 	constexpr const inline V &fetch(const K &) const {
-		return this->missing_key();
+		// A key is not in the map.
+		// This function should never be called. If it is called,
+		// you requested a nonexisting key.
+		return *static_cast<const V*>(nullptr);
```

Um, I just realized that we can just throw an exception. Nice error at runtime and "throw ABC non-constant expression" erro at compile time, understandable enough.